packages/dx-chart-core/src/plugins/animation/computeds.ts

```diff
@@ -41,7 +41,7 @@ const getPieAnimationName = () => {
 
 const getDefaultAreaAnimationOptions = () => '1s';
 
-const getDefaultPieAnimationOptions = ({ index }: Point) => `${0.7 + index * 0.1}s`;
+const getDefaultPieAnimationOptions = ({ index }: {index: number}) => `${0.7 + index * 0.1}s`;
```

Comment:
Why is it changed?

Reply:
There is ts error: missing the extra properties from type `Point` argument, value, color.

Reply:
It has to be fixed some other way then as function argument is actual of `TransformedPoint` type.